cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/update/ChangelogUpdateTest.java

```diff
@@ -123,6 +152,10 @@ private void makeNetworkChangesVariantSpecific(Network network) {
         load.setP0(newP).setQ0(newQ).getTerminal().setP(newP).setQ(newQ);
     }
 
+    private void makeRemoveChanges(Network network) {
```

Comment:
this method (and other utilitaries test methods) can be static

Reply:
Done